content/docs/installation/supported-releases.md

```diff
@@ -220,12 +221,12 @@ As of 2022-04-06, our testing coverage is:
 
 |      Vendor       | Oldest Kubernetes Release\*  |               Other Older Kubernetes Releases                 |
 |:-----------------:|------------------------------|---------------------------------------------------------------|
-|    [EKS][eks]     | 1.19 (EOL Jun 2022)          | 1.20 (EOL Sep 2022), 1.21 (EOL Feb 2023), 1.22 (EOL May 2023) |
-|    [GKE][gke]     | 1.19 (EOL Jun 2022)          | 1.20 (EOL Aug 2022), 1.21 (EOL Mar 2023), 1.22 (EOL Apr 2023) |
-|    [AKS][aks]     | 1.21 (EOL Jul 2022)          | 1.22 (EOL Nov 2022)                                           |
-| [OpenShift 4][os] | 1.19 (4.6 EUS, EOL Dec 2022) | 1.20 (4.7, EOL Nov 2022), 1.21 (4.8, EOL Feb 2023, EUS after) |
+|    [EKS][eks]     | 1.20 (EOL Nov 2022)          | 1.21 (EOL Feb 2023), 1.22 (EOL May 2023)                      |
+|    [GKE][gke]     | 1.20 (EOL Aug 2022)          | 1.21 (EOL Dec 2022), 1.22 (EOL Mar 2023)                      |
+|    [AKS][aks]     | 1.22 (EOL Nov 2022)          |                                                               |
+| [OpenShift 4][os] | 1.19 (4.6 EUS, EOL Dec 2022) | 1.20 (4.7, EOL Aug 2022), 1.21 (4.8, EOL Jan 2023, EUS after) |
```

Comment:
note: these are based on my quick readings of the various vendor support pages. Some of the changes are surprising to me (e.g. openshift reducing the support period for 4.7 and AKS being _so_ aggressive)

I have a half-finished piece of code which would autogenerate this table based on https://endoflife.date/ and other data sources, but haven't got round to finishing it :cry: